src/components/NodeDetailCard.js

```diff
@@ -34,9 +37,14 @@ export class NodeDetailCard extends React.Component {
           <span aria-hidden="true">&times;</span>
         </button>
         <div className="card-body">
-          <h5 className="card-title">{this.state.nodeData.kind.replace('_', ' ')}</h5>
+          <h4 className="card-title">{this.state.nodeData.kind.replace('_', ' ')}</h4>
+          <h5 className="card-subtitle">{this.state.nodeData.properties.name}</h5>
           <div className="card-text node-info-text">
-            <pre>{JSON.stringify(this.state.nodeData, null, 2)}</pre>
+            <pre>{JSON.stringify(this.state.nodeData.properties, function (k, v) {
+              if (k !== 'name') {
```

Comment:
Quite suggestive note, but consider replace this simply conditional statement with [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator). It is a great space saving :smiley: 

```suggestion
              k !== 'name' ? return v : null;
```

Reply:
After checking I saw that ternary operator doesn't have a sense in this case

Reply:
Yep those two approaches do not lead to the same result